cmd/rancher_release/check_rancher_rc_deps.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func checkRancherRCDepsCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "check-rancher-rc-deps",
+		Usage: "check if the Rancher version specified by the commit or pre-release title does not contain development dependencies and rc tags",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "commit",
+				Aliases:  []string{"c"},
+				Usage:    "last commit for a final rc",
+				Required: true,
+			},
+			&cli.StringFlag{
```

Comment:
StringFlag supports defining a default value for the help message. Also, checkRancherRCDeps might be a better place to set defaults for org and repo, rather than in the rancher package.